css/style.css

```diff
@@ -614,23 +622,30 @@ button.button-inline:hover {
 
 .colorselect {
 	overflow: hidden;
-	clear: both;
-	padding-top: 4px;
-	padding-left: 4px;
+	border-radius:3px;
+	flex-direction: row;
+	min-width: 240px;
```

Comment:
Not sure about that, since we have 2 places with different width using this.

![2017-06-08-230340_1155x200_scrot](https://user-images.githubusercontent.com/3404133/26950720-df736fbe-4c9e-11e7-9c47-91d26ed72c25.png)